instruments/nas/get.go

```diff
@@ -2,11 +2,15 @@ package nas
 
 import (
 	"encoding/json"
-
```

Comment:
Don't remove this line. This is the convention on how to order imports in go

Reply:
👍